README.md

```diff
@@ -1,11 +1,34 @@
 # react-scroll-sync
-Synced scroll position across multiple scrollable elements
 
-## Demo
+Synced scroll position across multiple scrollable elements.
 
-http://react-sync-scroll.netlify.com/
+## Installation
+
+```shell
+$ npm install react-scroll-sync
```

Comment:
Please add `--save` and yarn version.

Reply:
With the new version of npm, --save is optional as it automatically saves it as --save. It's useless in fact, except if you want to precise it's a devdep, and use --save-dev

Reply:
Okay for the yarn version even if I think the standard is still to use npm and people on yarn know easily to translate it into yarn way.

I add it most of time to mostly know the name of the package on npm registry.